### PR TITLE
Add instructions to build and test RBC

### PR DESCRIPTION
**Pull request by guilhermeleobas**
Probably depends on #526

### CODE REVIEW

File: rbc/heavydb/column_list.py [hunk 1]
```diff
@@ -91,6 +91,7 @@ def codegen(context, builder, signature, args):
 class ColumnList(Buffer):
     """
     RBC ``ColumnList<T>`` type that corresponds to HeavyDB COLUMN LIST
+    of arrays
```
Comment (pearu) [inline rbc/heavydb/column_list.py:94]: hmm, why arrays? It could be any of any support types.

File: rbc/heavydb/column_list_array.py [hunk 2]
```diff
@@ -30,6 +30,17 @@
 
 class ColumnListArray(ColumnList):
     """
+    RBC ``ColumnList<Array<T>>`` type that corresponds to HeavyDB COLUMN LIST
```
Comment (pearu) [inline rbc/heavydb/column_list_array.py:33]:
```suggestion
    RBC ``ColumnList<Array<T>>`` type that corresponds to HeavyDB COLUMN LIST of arrays.
```

File: rbc/heavydb/column_list.py (hunk 1 above)
Comment (pearu) [reply rbc/heavydb/column_list.py:94]: perhaps this was meant for ColumnListArray doc-string..

**Review by pearu — CHANGES_REQUESTED** [review]
I have a couple of nits, otherwise looks good!
Thanks, @guilhermeleobas!

File: rbc/heavydb/column_list.py (hunk 1 above)
Comment (guilhermeleobas) [reply rbc/heavydb/column_list.py:94]: Yes, ColumnListArray

**Review by pearu — APPROVED** [review]
LGTM! Thanks, @guilhermeleobas!

**Pull request merged**